test/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>$(SupportedNetTargets)</TargetFrameworks>
+    <TargetFrameworks>$(SupportedNetTargetsWithoutNet9)</TargetFrameworks>
```

Comment:
Instrumentaiton.StackExchangeRedis - support .NET9

Reply:
They have a lot of failures after adding `net9.0`. I think it's easier to manage to leave them out of this PR.